Sources/SwiftDocC/Model/Rendering/RenderNodeTranslator.swift

```diff
@@ -1852,23 +1852,41 @@ public struct RenderNodeTranslator: SemanticVisitor {
     private func variants(for documentationNode: DocumentationNode) -> [RenderNode.Variant] {
         let generator = PresentationURLGenerator(context: context, baseURL: bundle.baseURL)
         
-        return documentationNode.availableSourceLanguages
-            .sorted(by: { language1, language2 in
+        var allVariants: [SourceLanguage: [ResolvedTopicReference]] = Dictionary(uniqueKeysWithValues: documentationNode.availableSourceLanguages.map { ($0, [identifier]) })
```

Comment:
I don't think we've ever verified whether it holds that symbol graphs don't duplicate language representations per language. I don't want to rely on this behaviour, so I've reworked this to not expect uniqueness, in order to avoid any chances of crashing altogether.